paritytech/substrate-connect
# fix: prevent parachain chainspecs from being persisted

- The popup window asking for permission to persist chainspecs is now only applicable to relay chains, not parachain chainspecs.
- Parachains are no longer stored persistently.
- Note you can still **manually upload a parachain spec** to persist it which i think is reasonable.

## Testing
- Run the wallet template
- Open the demo project app
- The parachains should load in the demo project without opening any confirmation popup
- In the options page, the chainspecs table
- Try uploading a parachain chainspec from `packages/connect-known-chains/specs`
